torch/_dynamo/trace_rules.py

```diff
@@ -35,40 +40,2732 @@
 and trace_rules.lookup consolidation is done. Then the explicit listing of skip/inline objects have
 a higher priority, which can be used to override the skipfiles.check rules in some cases.
 """
-torch_name_rule_map = {
+manual_torch_name_rule_map = {
```

Comment:
We have two maps:
* ```manual_torch_name_rule_map```: manually added by PT developers.
* ```auto_torch_name_rule_map```: map generated by the heuristic in ```test/dynamo/test_trace_rules.py```, which was copied from ```allowed_functions.py```.

The functions in ```manual_torch_name_rule_map``` are functions with Dynamo implementation. If in the future, we hit cases that need Dynamo to support a new function, we probably should put them in this list.